cmd/create/cluster/runner.go

```diff
@@ -69,12 +102,29 @@ func (r *runner) run(ctx context.Context, _ *cobra.Command, _ []string) error {
 		}
 	}
 
+	var organization string
+	{
+		options := v1.ListOptions{
+			LabelSelector: fmt.Sprintf("%s=%s", "giantswarm.io/conformance-testing", "true"),
+		}
+		organizations, err := k8sClient.G8sClient().SecurityV1alpha1().Organizations().List(ctx, options)
```

Comment:
Would it make sense to use `controller-runtime` client as we generally use with new code these days?